streams/core.py

```diff
@@ -105,6 +105,23 @@ def emit(self, x):
                 result.append(r)
         return [element for element in result if element is not None]
 
+    def update(self, x, who=None):
+        self.emit(x)
+
+    def connect(self, parent):
+        ''' Connect another child to stream.
+            Note that parents go downstream and children go upstream.
+        '''
+        if self.parents == [None]:
```

Comment:
looks like it doesn't. I think I mirrored `if parent.children == [None]` below, which does happen. I will remove it.
Might be an issue later to address? Why does a stream with no parents contain `parents=[]` whereas a stream with no children contain `[None]`? (should it be `children = []`?)